package.json

```diff
@@ -19,9 +19,9 @@
     "chalk": "1.1.3",
     "cssnano": "3.10.0",
     "del": "2.2.2",
-    "electron": "1.7.2",
-    "execa": "0.6.3",
+    "electron": "1.7.3",
```

Comment:
Do you need to upgrade [`electron-builder`](https://github.com/electron-userland/electron-builder/releases) at the same time? 🤔